### PR TITLE
Add version option to spark_service.

### PR DESCRIPTION
**Pull request by jasonkuster**
Signed-off-by: Jason Kuster <jasonkuster@google.com>

Allows specifying version of Spark to use in managed Spark benchmarks.

### CODE REVIEW

**Comment by asaksena** [comment]
2 comments
- How are we planning to use this version information for a pkb managed cluster ? I guess this will be ignored in the interim ?
- If you have not tested it yet, I would like to make the change to the default config of other benchmarks that rely on spark service and make sure they continue to work

File: perfkitbenchmarker/configs/benchmark_config_spec.py [hunk 1]
```diff
@@ -203,6 +203,9 @@ def _GetOptionDecoderConstructions(cls):
             'default': spark_service.PROVIDER_MANAGED,
             'valid_values': [spark_service.PROVIDER_MANAGED,
                              spark_service.PKB_MANAGED]}),
+        'version': (option_decoders.EnumDecoder, {
+            'default': spark_service.SPARK_1_6_2,
```
Comment (amitsela) [inline perfkitbenchmarker/configs/benchmark_config_spec.py]: Spark runner upgraded to 1.6.3 - though it really shouldn't matter much, and would be compatible.

**Comment by s-deitz** [comment]
This PR is over a year old and it looks like we lost track of it. Is this still valid? If so, please reply and somebody will review in the next couple of weeks.

**Comment by jasonkuster** [comment]
I think we can close this. I can't remember why this was necessary and I've been working on other things for a while now.

**Pull request closed without merging**